### ISSUE TITLE
Fig not working on versions below 11.3

### ISSUE THREAD

**Issue by andresfdel17** [withfig/fig#2028]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 10.15.7 (19H2026)

### Expected behaviour

Fig autocomplete work correctly, and fig dashboard work correctly

### Actual behaviour

fig dashboard  is a white screen and fig autocomplete is not working anything

### Steps to reproduce

I updated to last version NO Beta

### Environment

```yaml
fig-details:
  - 2.7.7
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz
  - cores: 2
  - mem: 16.00 GB
os-info:
  - macOS 10.15.7 (19H2026)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/andresfdel13
  - exe-path: /usr/local/bin/fig
  - install-method: brew
  - env-vars:
    - DISPLAY: /private/tmp/com.apple.launchd.INuUTOHvpT/org.xquartz:0
    - FIGTERM_SESSION_ID: fd7694b2-9953-4581-8120-f563d6ab7cfe
    - FIG_HOSTNAME: andresfdel13-MacBook-Pro.local
    - FIG_TERM: 2.7.7
    - PATH: /Users/andresfdel13/.nvm/versions/node/v16.17.0/bin:/Users/andresfdel13/google-cloud-sdk/bin:/opt/local/bin:/opt/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/andresfdel13/.fig/bin:/Users/andresfdel13/.local/bin:/usr/local/mysql/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 1854
    - FIG_PARENT: fd7694b2-9953-4581-8120-f563d6ab7cfe
    - FIG_SET_PARENT: fd7694b2-9953-4581-8120-f563d6ab7cfe
```

**Comment by grant0417** [comment]
Hey @andresfdel17, could you click on the "Autocomplete Devtool" button and show the output of the console?

<img width="417" alt="Screenshot 2022-11-10 at 11 30 48 AM" src="https://user-images.githubusercontent.com/7355421/201188862-b609b978-c5de-4712-ad44-5659739780af.png">

**Comment by andresfdel17** [comment]
![Captura de Pantalla 2022-11-10 a la(s) 2 32 09 p  m](https://user-images.githubusercontent.com/53264064/201189201-352566e7-2f5c-41d9-8870-d85b0c606ac1.png)

**Comment by andresfdel17** [comment]
That icon i can´t see it

**Comment by andresfdel17** [comment]
That problem was in last update

**Comment by andresfdel17** [comment]
Continue the trouble, fig is not working

**Comment by grant0417** [comment]
Can you run `fig debug devtools autocomplete`

**Comment by andresfdel17** [comment]
![Captura de Pantalla 2022-11-10 a la(s) 3 41 55 p  m](https://user-images.githubusercontent.com/53264064/201201677-3c9fbff2-211d-405f-b0ba-594b067783a1.png)

**Comment by grant0417** [comment]
Can you try typing into the console at the bottom: `window.location.href = "https://autocomplete.fig.io/"`

**Comment by andresfdel17** [comment]
OK, give me a minute

**Comment by andresfdel17** [comment]
![Captura de Pantalla 2022-11-10 a la(s) 4 02 27 p  m](https://user-images.githubusercontent.com/53264064/201205217-7744e2a2-14bc-4d45-a371-eddfbe7b2e0d.png)

**Comment by grant0417** [comment]
Not sure what is going on here, it seems like the page is not loading, after you set the page it should look something like this:

<img width="987" alt="Screenshot 2022-11-10 at 1 09 31 PM" src="https://user-images.githubusercontent.com/7355421/201206467-09c3c85b-4193-4365-b2a9-81d3cca66b3f.png">

**Comment by grant0417** [comment]
Can you go to `https://autocomplete.fig.io` in your browser and see if the page loads there?

**Comment by andresfdel17** [comment]
![Captura de Pantalla 2022-11-10 a la(s) 4 11 18 p  m  (2)](https://user-images.githubusercontent.com/53264064/201206790-5755f453-64aa-4ec7-8bdb-c36f8267f630.png)

**Comment by grant0417** [comment]
Hmm, seems like it did load properly in your browser, are you able to quit fig and run `FIG_LOG_LEVEL=info /Applications/Fig.app/Contents/MacOS/fig_desktop` and share the logs from there

**Comment by TofuTseng** [comment]
same issue here.

**Comment by grant0417** [comment]
If either of you are available for a debugging call here is a link to schedule one: https://calendly.com/grant-fig/debugging

**Comment by keisei77** [comment]
+1

**Comment by andresfdel17** [comment]
![Captura de Pantalla 2022-11-11 a la(s) 9 23 08 a  m](https://user-images.githubusercontent.com/53264064/201359735-e1d743cd-4b32-47f6-8921-172743add254.png)
![Captura de Pantalla 2022-11-11 a la(s) 9 23 11 a  m](https://user-images.githubusercontent.com/53264064/201359753-e1611224-9465-48d5-9ed5-85a8cf1e8f49.png)

**Comment by andresfdel17** [comment]
For the meeting, i´m sorry. I work at home all day. I can´t join that meeting

**Comment by glleroux** [comment]
This is happening to me too — white screen + no autocomplete. Could it be the current version of Fig (2.7.7) being incompatible with with my Mac OS version (11.1)?

**Comment by o4suki** [comment]
+1

**Comment by glleroux** [comment]
+1

**Comment by arb85** [comment]
+1

**Comment by CassianoThurow** [comment]
+1

**Comment by grant0417** [comment]
Just want to say once again, if you are having this issue and are available here is a link to schedule a debugging call: https://calendly.com/grant-fig/debugging

**Comment by grant0417** [comment]
So it seems this is an issue with use using an API that requires macOS 11.3 or higher, working on a fix that will bring back support to lower versions!

**Comment by grant0417** [comment]
This should be fixed on version 2.7.9

You can update by running `fig update`
